tests/functional/cypress/helpers/jwt_helper.js

```diff
@@ -0,0 +1,133 @@
+const KJUR = require('jsrsasign');
+const uuid = require('uuid/v1');
+const JSONWebKey = require('json-web-key');
+const jose = require('node-jose');
+const JWK = jose.JWK;
+const JWE = jose.JWE;
+
+const crypto = require('crypto');
+
+const signingKeyString = '-----BEGIN RSA PRIVATE KEY-----\n' +
```

Comment:
Can we not read these from files like we do with the current `jwt_helper`?
Also I don't get why we need 1 specifically for cypress?. We can just reuse the one we already have?

Reply:
Cypress doesn't have access to the fs module - so that's why these have been pulled in as strings. If you attempt to use the original jwt_helper, it will drop out failing to find the module. Creating a cypress version allows the two sets of tests to run alongside one another whilst we decide which approach is best.